### PR TITLE
Adds new documentation for Alpha validator

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Enhancement        | yes

### CODE REVIEW

File: docs/book/validators/alpha.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Alpha
+
+`Laminas\I18n\Validator\Alpha` allows you to validate if a given value
+**contains only alphabetical characters**. There is no length limitation for the
+input you want to validate. This validator is identical to the
+`Laminas\I18n\Validator\Alnum` validator with the exception that it does not
```
Comment (michalbundyra) [inline docs/book/validators/alpha.md]: should we link to the documentation of the Alnum validator here?

File: docs/book/validators/alpha.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Basic Usage
+
+```php
+$validator = new Laminas\I18n\Validator\Alpha();
+
+if ($validator->isValid('Abcd')) {
+    // value contains only allowed chars
+} else {
+    // false
+}
```
Comment (michalbundyra) [inline docs/book/validators/alpha.md]: maybe better provide examples as in other validators documentation:
```php
var_dump($validator->isValid('Abcd')); // true
var_dump($validator->isValid('Virus24')); // false
```

File: docs/book/validators/alpha.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+instantiation or afterwards via the `setAllowWhiteSpace()` method.
+
+```php fct_label="Constructor Usage"
+$validator = new Laminas\I18n\Validator\Alpha(['allowWhiteSpace' => true]);
+
+if ($validator->isValid('Abcd and efg')) {
+    // value contains only allowed chars
+} else {
+    // false
+}
```
Comment (michalbundyra) [inline docs/book/validators/alpha.md]: The same about examples as above

File: docs/book/validators/alpha.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+The default value of this option is `false` that means whitespace characters are
+not allowed.
+
+## Using different Languages
+
+When using `Laminas\I18n\Validator\Alpha`, the language provided by the user's
+browser will be used to set the allowed characters. For locales outside of
+English, this means that additional alphabetic characters may be used
+&mdash; such as `ä`, `ö` and `ü` from the German alphabet.
```
Comment (michalbundyra) [inline docs/book/validators/alpha.md]: should we use normal `-` here?

File: docs/book/validators/alpha.md (hunk 2 above)
Comment (froschdesign) [reply docs/book/validators/alpha.md]: I'm still not sure if I like the `var_dump` here. Other options are:

```php
$validator->isValid('Abcd'); // true
```
```php
$result = $validator->isValid('Abcd'); // true
```
```php
if ($validator->isValid('Abcd')) {
    // value contains only allowed chars
}
```
…

File: docs/book/validators/alpha.md (hunk 4 above)
Comment (froschdesign) [reply docs/book/validators/alpha.md]: I think we should remove the dash for simplification and the different meanings / handlings in languages. See also: "[En dash versus em dash](https://en.wikipedia.org/wiki/Dash#En_dash_versus_em_dash)"

File: docs/book/validators/alpha.md (hunk 2 above)
Comment (michalbundyra) [reply docs/book/validators/alpha.md]: ok, maybe if statement is the best then, but I guess we should remove `else { ...`

Comment (froschdesign) [reply docs/book/validators/alpha.md]: > maybe if statement is the best

I quite agree.

**Comment by michalbundyra** [comment]
Thanks, @froschdesign!

**Pull request closed without merging**